docs/2022/puzzles/day01.md

```diff
@@ -1,11 +1,75 @@
 import Solver from "../../../../../website/src/components/Solver.js"
 
 # Day 1: Calorie Counting
+by [@bishabosha](https://twitter.com/bishabosha)
 
 ## Puzzle description
 
 https://adventofcode.com/2022/day/1
 
+## Solution Summary
+
+First transform the input into a `List` of `Inventory`, each `Inventory` is a list of `Int`, representing the calorie
+count of an item in the inventory, this is handled in `scanInventories`.
+
+### Part 1
+
+given the `List` of `Inventory`, we must first find the total calorie count of each inventory.
+
+> For a single `Inventory`, we do this with the `sum` method its `items` property (found in the `List` class). e.g. `inventory.items.sum`.
```

Comment:
Why is this in a quote `>` ?

Also I think there is something missing in the sentence:
```suggestion
For a single `Inventory`, we do this using the `sum` method on its `items` property (found in the `List` class). e.g. `inventory.items.sum`.
```

Reply:
thanks